ui-components/autocompleter.md

```diff
@@ -26,7 +26,7 @@ The two arguments this method take are:
 | Name | Value | Requirement | Description |
 | ---- | ----- | ----------- | ----------- |
 | ch | string | Required | The character to trigger the autocompleter. |
-| fetch |  (pattern: string, maxResults: number) => Promise<AutocompleterItem[]> | Required | A function that is passed the current matched text pattern and the maximum number of expected results. The function should return a promise containing matching results. |
+| fetch |  (pattern: string, maxResults: number, fetchOptions: Record<string, any>) => Promise<AutocompleterItem[]> | Required | A function that is passed the current matched text pattern and the maximum number of expected results. The function should return a promise containing matching results. |
```

Comment:
Probably best to note `fetchOptions` is optional 🤔

Reply:
It's not optional? it's defaulted and would just be an empty object. However yeah we probably should clarify it'll be the value passed from reload, otherwise will be an empty object.

Reply:
Oh right sorry I was looking at it backwards

yeah, defaulted seems good

Reply:
I did some re-organizing and added a sentence about the defaults and how fetchOptions is passed from reload, so let me know what you think :)